cache/cachetest/cacher.go

```diff
@@ -23,11 +23,11 @@ var Tests = []struct {
 	Size int
 	Func func(t *testing.T, c cache.Cacher[ids.ID, int64])
 }{
-	{Size: 1, Func: TestBasic},
-	{Size: 2, Func: TestEviction},
+	{Size: 1, Func: Basic},
+	{Size: 2, Func: Eviction},
 }
 
-func TestBasic(t *testing.T, cache cache.Cacher[ids.ID, int64]) {
```

Comment:
Why are these renamed? Is this because we felt the `cachetest.Test*` is stuttering?

Reply:
Yes